src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -742,6 +742,8 @@ async def _handle_message(self, message: RTVIMessage):
                 case "update-config":
                     update_config = RTVIUpdateConfig.model_validate(message.data)
                     await self._handle_update_config(message.id, update_config)
+                case "disconnect-bot":
```

Comment:
@aconchillo the use case for this message is for a client to instruct the bot to leave the call. I think pushing an `EndFrame()` is appropriate. We could alternatively call `leave()` on the Daily transport, but `EndFrame()` feels like the more appropriate usage. Any preference?

@Regaddi we might want this name to be more about leaving than disconnecting. The idea is that once this message is passed, the pipeline will stop.

Reply:
I think it depends on the desired possibilities after this message. Should it be possible the resurrect the pipeline? If so, should that be possible through a separate message?

Reply:
EndFrame is the most suitable frame. Description:
```
Indicates that a pipeline has ended and frame processors and pipelines
    should be shut down. If the transport receives this frame, it will stop
    sending frames to its output channel(s) and close all its threads. Note,
    that this is a control frame, which means it will received in the order it
    was sent (unline system frames).
```

An EndFrame stops the pipeline. There isn't an option to disconnect the bot but leave the pipeline running. The bot and pipeline are synonymous. Despite that, maybe `disconnect-bot` is still a good option. WDYT?